lib/rules/indent.js

```diff
@@ -556,49 +556,6 @@ module.exports = {
         }
 
         /**
-         * Check indentation for blocks
-         * @param {ASTNode} node node to check
-         * @returns {void}
-         */
-        function addBlockIndent(node) {
-
-            let blockIndentLevel;
-
-            if (node.parent && isOuterIIFE(node.parent)) {
-                blockIndentLevel = options.outerIIFEBody;
-            } else if (node.parent && (node.parent.type === "FunctionExpression" || node.parent.type === "ArrowFunctionExpression")) {
-                blockIndentLevel = options.FunctionExpression.body;
-            } else if (node.parent && node.parent.type === "FunctionDeclaration") {
-                blockIndentLevel = options.FunctionDeclaration.body;
-            } else {
-                blockIndentLevel = 1;
-            }
-
-            /*
-             * If the block starts on its own line, then match the tokens in the block against the opening curly of the block.
-             * Otherwise, match the token in the block against the tokens in the block's parent.
-             *
-             * For example:
-             * function foo() {
-             *   {
-             *      // (random block, tokens should get matched against the { that opens the block)
-             *      foo;
-             *   }
-             *
-             * if (foo &&
-             *     bar) {
-             *     baz(); // Tokens in the block should get matched against the `if` statement, even though the opening curly is indented.
-             * }
-             */
-            const tokens = getTokensAndComments(node);
-            const tokenToMatchAgainst = tokenInfo.isFirstTokenOfLine(tokens[0]) ? tokens[0] : sourceCode.getFirstToken(node.parent);
-
-            offsets.matchIndentOf(tokenToMatchAgainst, tokens[0]);
-            offsets.setDesiredOffsets(tokens, tokens[0], blockIndentLevel);
-            offsets.matchIndentOf(tokenToMatchAgainst, tokens[tokens.length - 1]);
```

Comment:
This function was moved to a different location in the file because it would cause `no-use-before-define` errors otherwise. However, only the last 5 lines have been changed.